### PR TITLE
User profile page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
acc pge

### CODE REVIEW

File: profile/profile.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    margin-bottom: 3rem;
+}
+.para-md{
+    padding: 1rem;
+}
+.profile-details-main div .paragraph-md {
+    padding-bottom: 1rem;
+}
+.logout-btn{
+    display: flex;
```
Comment (praveenreddy1798) [inline profile/profile.css:29]: can have a utility class like flex-vertical and use it as you will use this at multiple places

File: reset.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.uppercase{
+    text-transform: uppercase;
+}
+.bolder{
+    font-weight: bolder;
+}
```
Comment (praveenreddy1798) [inline reset.css:53]: Try to not use reset.css in as much as possible in your projects. You can use it from the one you set up in your component library. Use it only if you need somethings to be overwritten.

I see some classes like w-fit, w-full in your reset. This are suppose to come in the utils. Only the ones that come by default from the browser and if you need to override then use in reset

**Review by harshkotkar5557 — APPROVED** [review]
clean code and looks good to me

**Review by manojasarsa — APPROVED** [review]
Looks good!

File: reset.css (hunk 2 above)
Comment (vaibhav18matere) [reply reset.css:53]: i will move those reset.css files in component library once i am done with this project

**Pull request merged**